### PR TITLE
[4.0] Fix JS error in searchtools

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #22784

### Summary of Changes

Only perform actions on the `#sort` element if it exists

### Testing Instructions

Refer to issue

### Expected result

No JS error

### CODE REVIEW

File: build/media_src/system/js/searchtools.es6.js [hunk 1]
```diff
@@ -456,14 +456,16 @@
 
     const sort = document.getElementById('sorted');
 
-    if (sort.hasAttribute('data-caption')) {
-      const caption = sort.getAttribute('data-caption');
-      document.getElementById('captionTable').textContent += caption;
-    }
+    if (sort) {
+      if (sort.hasAttribute('data-caption')) {
```
Comment (dgrammatiko) [inline build/media_src/system/js/searchtools.es6.js]: `if (sort && sort.hasAttribute('data-caption')) {`
It's a one liner one if statement same result

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

File: build/media_src/system/js/searchtools.es6.js [hunk 2]
```diff
@@ -456,14 +456,16 @@
...
+        const caption = sort.getAttribute('data-caption');
+        document.getElementById('captionTable').textContent += caption;
+      }
 
-    if (sort.hasAttribute('data-sort')) {
-      const ariasort = sort.getAttribute('data-sort');
-      sort.parentNode.setAttribute('aria-sorted', ariasort);
+      if (sort.hasAttribute('data-sort')) {
```
Comment (dgrammatiko) [inline build/media_src/system/js/searchtools.es6.js]: Same here:
` if (sort && sort.hasAttribute('data-sort')) {`

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

File: build/media_src/system/js/searchtools.es6.js (hunk 2 above)
Comment (C-Lodder) [reply build/media_src/system/js/searchtools.es6.js]: Makes no odds really, wrapping it all in 1 conditional statement or using it twice in each statement.

**Comment by laoneo** [comment]
Thanks

**Pull request merged**